misc/acrn-config/launch_config/com.py

```diff
@@ -419,14 +419,15 @@ def dm_arg_set(names, sel, dm, vmid, config):
 
     # uuid get
     scenario_uuid = launch_cfg_lib.get_scenario_uuid()
+    sos_vmid = launch_cfg_lib.get_sos_vmid()
 
     # clearlinux/android/alios
-    dm_str = 'acrn-dm -A -m $mem_size $boot_GVT_option"$GVT_args" -s 0:0,hostbridge -s 1:0,lpc -U {}'.format(scenario_uuid[vmid])
+    dm_str = 'acrn-dm -A -m $mem_size $boot_GVT_option"$GVT_args" -s 0:0,hostbridge -U {}'.format(scenario_uuid[vmid + sos_vmid])
     if uos_type in ("CLEARLINUX", "ANDROID", "ALIOS"):
         if uos_type == "CLEARLINUX":
-            print("{} \\".format(dm_str), file=config)
+            print("{} -s 1:0,lpc \\".format(dm_str), file=config)
         else:
-            print('{} $npk_virt \\'.format(dm_str), file=config)
+            print('{} -s 1:0,lpc $npk_virt \\'.format(dm_str), file=config)
```

Comment:
why npk_virt must follow lpc immediately?

Reply:
npk_virt follow lpc is just for coding convenience, i will update it.